tests/integration/targets/inventory/inventory1.yml

```diff
@@ -0,0 +1,6 @@
+---
+plugin: cloud.terraform.inventory
+
+binary_path: 
```

Comment:
Why do you have an empty binary path everywhere? The config item should be completely optional.

Reply:
deleted from the files